src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Planta.kt

```diff
@@ -15,7 +17,9 @@ class Menta(anioObtencionSemilla: Int, altura: Float) : Planta(anioObtencionSemi
   override fun daSemillas() = this.esFuerte() || altura > 0.4
```

Comment:
Acá faltó:

### Redundancia mínima

- `Planta.daSemillas` -> La primera parte (`this.esFuerte()`) está repetida en todas las subclases.